### DISCUSSION TITLE
Inventory app added to examples

### DISCUSSION THREAD

**Discussion by JaapWijnen** [pointfreeco/swift-composable-architecture#1631] [Ideas]
I would like to suggest adding the Inventory app (that's being used for the SwiftUI navigation series) to also be added to the TCA examples. It showcases some more complex uses of identified arrays where selected items are editable etc and shows a few ways to do navigation. I think it would really be a good addition since there currently aren't any examples that showcase this behaviour (mainly the selection and navigation to that selection which can then be edited) using TCA. 
I'd love to set it up if there's interest from @stephencelis @mbrandonw to add it to the list.

**Comment by stephencelis** [comment]
@JaapWijnen We're in the process of refining navigation-specific tools in TCA, but they're not ready quite yet. Once they are, the Inventory app will be a natural demo for us to add. :smile:

**Reply by JaapWijnen** [reply]
@stephencelis I totally understand! would love to give it a try with the current tools though! (I understand that the navigation part will be partly different than the SwiftUI version) let me know if thats interesting or a potential waste of time in your eyes! :)

**Reply by stephencelis** [reply]
@JaapWijnen I think we'll be exploring it ourselves soon, so if you don't want to duplicate that work it's totally something you can avoid. If you merely want to explore it for your own understanding, we're not going to dissuade you though!

**Reply by JaapWijnen** [reply]
Yeah it would be for my own understanding as well! If id make a PR it might also be helpful as a guide for moving from that to the new tools that you guys are working on!

**Reply by stephencelis** [reply]
Definitely feel free to share a branch here! I think we'll probably hold off on accepting any case study PRs right now, but we'd still be interested to see what you come up with.

**Comment by JaapWijnen** [comment]
@stephencelis I made a PR to showcase the TCA version of the Inventory app. Let me know what you think! Any feedback is appreciated!

**Comment by JaapWijnen** [comment]
I've made a TCA version of the swiftui-navigation Inventory app which can be found in this (closed) PR https://github.com/pointfreeco/swift-composable-architecture/pull/1650
